script/dependency_install.sh

```diff
@@ -15,8 +15,8 @@ function install_python()
     curl -L https://raw.githubusercontent.com/pyenv/pyenv-installer/master/bin/pyenv-installer | bash
     pyenv install -v 3.8.10
     pyenv global 3.8.10
-    cd external/zenoh_carla_bridge/carla_agent && poetry env use $(pyenv which python)
-    cd external/zenoh_autoware_v2x && poetry env use $(pyenv which python) && poetry install --no-root
+    cd $HOME/autoware_carla_launch/external/zenoh_carla_bridge/carla_agent && poetry env use $(pyenv which python)
```

Comment:
Using `$HOME` here is not flexible. If we don't have the folder on the same path, then developers need to update the code here.
I suggest to use `$AUTOWARE_CARLA_ROOT` instead. You can find it in `env.sh`

Reply:
Sure! I have replaced "$HOME/autoware_carla_launch" with "${AUTOWARE_CARLA_LAUNCH}" in the latest commit. Please check it when you have time.